docs/Training-ML-Agents.md

```diff
@@ -446,7 +466,7 @@ Below is a list of included `sampler-type` as part of the toolkit.
   - **sub-arguments** - `intervals`
 
 The implementation of the samplers can be found at
-`ml-agents-envs/mlagents_envs/sampler_class.py`.
+`ml-agents/mlagents/trainers/sampler_class.py`.
```

Comment:
Note: this is likely wrong in the release docs as well.

Reply:
Can you link to the file so that we get some automated checks on it?

Reply:
Done